src/Features/Core/Portable/AddParameter/AbstractAddParameterCodeFixProvider.cs

```diff
@@ -222,7 +224,8 @@ private int NonParamsParameterCount(IMethodSymbol method)
             }
             else
             {
-                var name = semanticFacts.GenerateNameForExpression(semanticModel, expression);
+                var name = semanticFacts.GenerateNameForExpression(
+                    semanticModel, expression, capitalize: false, cancellationToken: cancellationToken);
```

Comment:
i see.  Technically we likely want to use some sort of naming service that can follow user preferences.  However, in practice, it's highly unlikely that users are going to do anything special with parameters, so i think it's fine to keep the current behavior (which is to make a camel-cased name).